### PR TITLE
add site-footer

### LINKED ISSUE #4: Add: Site Footer
Is your Feature Request Related to a Problem/a Blocker? Please Describe.
N/A

--------

Describe How the Solution Should Behave
The footer component should be displayed on all pages of the website, the component will be in the components directory in the src directory and imported into the app.js. All the links in the footer should reflect real pages on the website.

---------

Screenshots of the Design/if Any

---------

Additional context
Formik for input validation.
react-router-dom is already imported to the repo.
All routes should be a constant variable stored in the src/routes.js file and exported from there for ease of use.

### PR DESCRIPTION
**Pull request by bayogluteoman**

### CODE REVIEW

**Review by Iseluin — COMMENTED** [review]
For future reference, please add "Closes#12(number of the issue)" in your pull requests so that it closes the linked issue automatically. I will check the live site on my local machine and add my review on the footer asap.

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        <div className="text-center pt-2 pb-5 bg-[#6A2C70]">
+          <span className="text-white font-semibold">
+            © 2021-2022 Copyright Bursapediary Inc.
+          </span>
+        </div>
+      </footer>
+    </div>
+  );
+}
+
```
Comment (Iseluin) [inline src/components/Footer.jsx]: Please adjust your code so that the icons and "copyright" section looks like the original figma design.

**Review by Iseluin — CHANGES_REQUESTED** [review]
Good work Teoman! Please adjust the small things I wrote below and we are good to go.

File: src/components/Footer.jsx (hunk 1 above)
Comment (Iseluin) [reply src/components/Footer.jsx]: After you change the icons to react-icons library and fix the icon placement, please be sure to add:

- unit & snapshot tests.
- Change the paragraphs to something relevant with Bursapediary.

Then we will be good to merge this. Good work!

Comment (bayogluteoman) [reply src/components/Footer.jsx]: I changed the paragraphs and social media icon's location and I add hover for change icon's color. Also I add home icon for locations. Lastly, couldn't manage tests.

- 
image

Comment (Iseluin) [reply src/components/Footer.jsx]: Good work!

File: src/App.jsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
-import './App.css';
+import Footer from './components/Footer';
```
Comment (Iseluin) [inline src/App.jsx]: My mistake I skipped this on the previous review, can we put the component in its own folder like src/components/Footer/<filename>

File: src/index.js [hunk 3]
```diff
@@ -1,16 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import { Provider } from 'react-redux';
 import App from './App';
-import store from './app/store';
 import * as serviceWorker from './serviceWorker';
```
Comment (Iseluin) [inline src/index.js]: Global imports at the top, a blank line and local imports at the bottom please.

**Review by Iseluin — CHANGES_REQUESTED** [review]
Thanks for the alterations. I added 2 minor things I missed on the last review. Please add the tests too and we can merge this.

File: src/App.jsx (hunk 2 above)
Comment (bayogluteoman) [reply src/App.jsx]: It gives me this eslint error 
src\App.jsx
  Line 1:20:  Useless path segments for "../src/components/Footer/Footer", should be "./components/Footer/Footer"  import/no-useless-path-segments
But I add Footer folder.

File: src/components/Footer/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+function Footer() {
+  return (
+    <div>
+      <footer
+        className="text-center lg:text-left bg-[#6A2C70]"
+        data-testid="valid-footer"
+      >
+        <div className="mx-6 pt-5 text-center md:text-left">
+          <div className="grid grid-1 md:grid-cols-3 lg:grid-cols-3 gap-12">
+            <div className="">
```
Comment (Iseluin) [inline src/components/Footer/Footer.jsx]: You can delete the className attribute if we are not going to use it.

File: src/components/Footer/Footer.jsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+              </div>
+
+              <div className="flex justify-start">
+                <AiFillHome className="text-[#ffffff] text-xl mr-1" />
+                <a href="https://www.google.com/maps/place/1178+Broadway+2+1033,+New+York,+NY+10001,+Amerika+Birle%C5%9Fik+Devletleri/@40.7452728,-73.9905456,17z/data=!3m1!4b1!4m5!3m4!1s0x89c259a60f737d9f:0x29524ecce6633e4e!8m2!3d40.7452688!4d-73.9883569">
+                  <p className="flex items-center justify-center md:justify-start mb-4 text-white">
+                    1178 Broadway, Floor 2, #1033, New York, 10001, US
+                  </p>
+                </a>
+              </div>
```
Comment (Iseluin) [inline src/components/Footer/Footer.jsx]: These adresses and the icons are not responsive when the viewport width is < 768, they are aligned to the start but they must be centered.

**Review by Iseluin — CHANGES_REQUESTED** [review]
Good work @bayogluteoman , I just encountered some unused classnames and a minor responsivity issue. Please alter them.

**Review by Iseluin — APPROVED** [review]

**Pull request merged**